src/crimson/os/seastore/omap_manager/btree/string_kv_node_layout.h

```diff
@@ -786,14 +786,15 @@ class StringKVInnerNodeLayout {
   static std::string balance_into_new_nodes(
     const StringKVInnerNodeLayout &left,
     const StringKVInnerNodeLayout &right,
+    uint32_t pivot_idx,
     StringKVInnerNodeLayout &replacement_left,
     StringKVInnerNodeLayout &replacement_right)
   {
+    ceph_assert(!left.below_min() || !right.below_min());
     uint32_t left_size = omap_inner_key_t(left.get_node_key_ptr()[left.get_size()-1]).key_off;
     uint32_t right_size = omap_inner_key_t(right.get_node_key_ptr()[right.get_size()-1]).key_off;
     uint32_t total = left_size + right_size;
     uint32_t pivot_size = total / 2;
-    uint32_t pivot_idx = get_balance_pivot_idx(left, right);
```

Comment:
IIUC we won't tolerate underflow nodes in the fix-kv-btrees.

So the commit message, e.g.:
```
crimson/os/seastore: adjust and fix rebalance logic in lba/backref/omap tree
* For fixed_kv_btree, drop `prefer_left`.
* For omap btree, tolerate underflow nodes if rebalance has the same results.

Fixes: https://tracker.ceph.com/issues/71307
```

Reply:
Fixed.